libp2p/security/simple/transport.py

```diff
@@ -30,6 +30,10 @@ def __init__(self, local_key_pair: KeyPair, key_phrase: str) -> None:
             )
 
         session = InsecureSession(self, conn, ID(b""))
+        # TODO: Calls handshake to make them know the peer id each other, otherwise tests fail.
```

Comment:
```suggestion
        # TODO: Calls handshake to make them know the peer id of each other, otherwise tests fail.
```
Question:
- What do you mean by `call handshake`?
- Does test fail now because we don't `call handshake` here? or does test succeed now because we `await session.run_handshake()`?

Reply:
1. I mean calling `InsecureSession.run_handshake`.
2. The test passes now because we `await session.run_handshake()`. I will rephrase the sentence.